content/300-guides/050-application-lifecycle/100-developing-with-prisma-migrate/100-advanced-migrate-scenarios.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: Advanced migration scenarios
+---
+
+<TopBlock>
+
+* Create a draft
```

Comment:
Is this intended? It looks odd or maybe I just don't get it.

Reply:
Just draft